linux_os/guide/services/ssh/ssh_server/sshd_set_idle_timeout/bash/shared.sh

```diff
@@ -1,5 +1,8 @@
-# platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+# platform = multi_platform_all
+
 . /usr/share/scap-security-guide/remediation_functions
+
 {{{ bash_instantiate_variables("sshd_idle_timeout_value") }}}
 
-replace_or_append '/etc/ssh/sshd_config' '^ClientAliveInterval' $sshd_idle_timeout_value '@CCENUM@' '%s %s'
+{{{ bash_sshd_config_set("ClientAliveInterval", "$sshd_idle_timeout_value") }}}
+{{{ bash_sshd_config_set("ClientAliveCountMax", "0") }}}
```

Comment:
ClientAliveCountMax needs to maintain itself in a new rule. One configuration change per rule only.